manual/alshelp.css

```diff
@@ -0,0 +1,68 @@
+#body {
+overflow-y: scroll;
+overflow-x: hidden;
+}
+
+#left_top {
+position: absolute;
+top:0;
+left:0;
+height:8em;
+width:13em;
+background-color:ivory;
+}
+
+#left_bottom {
+position: absolute;;
+top:8em;
+left:0;
+height:100%;
+width:13em;
+background-color:cornsilk;
+}
+
+#right {
+position: absolute;;
+top:0;
+left:13em;
+height:120%;
```

Comment:
The 120% height and width setting causes problem as you resize the window. (sometime the "right" frames scroll bar is visible, sometime hidden, etc).

I'd recommend removing the height/width and then setting top, right, bottom to 0 (keeping left at 13em).